src/foam/core/AbstractFObjectArrayPropertyInfo.java

```diff
@@ -34,20 +34,26 @@ public Object fromString(String value) {
   public Object fromXML(X x, XMLStreamReader reader) {
     List objList = new ArrayList();
     String startTag = reader.getLocalName();
-
+    Class defaultClass = this.getClassInfo().getObjClass();
+    Class<?> clazz;
+    try {
+       clazz = Class.forName(this.of());
```

Comment:
Is there a better way to do that?
the this.of() returns string name of the parent class so I can't use x.create()